foxbroadcasting/react-select
# [DPP-925] Add unique select component

## What did you implement:
Implementing issue: [DPP-925](https://fng-jira.fox.com/browse/DPP-925)

## How did you implement it:
Add a new select type `unique` which includes new react logic and css styles

## Files and functions to review
Since I replaced all the tabs for spaces in `src/Select.js` Here is a table to help you with the review

|File|function|description|
|---|--------|-----------|
|`src/Select.js`|renderValue|line 745 - 749: Add place holder for unique select|
||renderOuter|line 1058 - 1059: Add `is-unique` class|
||renderUniqueSelect|line 1076 - 1121: Add the new select component| 
||render|line 1147: added the `is-unique` condition|
|||line 1164 - 1165: Added another `is-unique` condition|
|`src/Option.js`|render|line 83 - 90: render selected option|

## Screenshots
**Image**
![screen shot 2018-09-18 at 7 31 24 am](https://user-images.githubusercontent.com/1103543/45695037-6e0f3d80-bb15-11e8-9aa7-121b3052a305.png)


**GIF**

![iktkbhubub](https://user-images.githubusercontent.com/1103543/45695115-98f99180-bb15-11e8-8195-b6e0b48a424f.gif)


## How can we verify it:
Check the steps for the fng-cms [PR](https://github.com/foxbroadcasting/fng-cms/pull/523)
